src/Validation/DatabasePresenceVerifier.php

```diff
@@ -43,8 +43,13 @@ public function getCount($collection, $column, $value, $excludeId = null, $idCol
      */
     public function getMultiCount($collection, $column, array $values, array $extra = [])
     {
-        // Generates a regex like '/(a|b|c)/i' which can query multiple values
-        $regex = '/('.implode('|', $values).')/i';
+        // Nothing can match an empty array. Return early to avoid matching an empty string.
+        if ($values === []) {
+            return 0;
+        }
+
+        // Generates a regex like '/^(a|b|c)$/i' which can query multiple values
+        $regex = new Regex('^('.implode('|', $values).')$', 'i');
```

Comment:
I'm aware it wasn't done before, but shouldn't each individual value be sent through `preg_quote`?

Reply:
Indeed they should. I'll add tests also.